src/main/java/de/dataelementhub/model/dto/datatransfer/DataTransfer.java

```diff
@@ -1,20 +1,16 @@
-package de.dataelementhub.model.dto.export;
+package de.dataelementhub.model.dto.datatransfer;
```

Comment:
I don't think "datatransfer" is a good package name here, since "dto" already is the parent package.

Reply:
Renamed to importexport